editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/AdvancedEndpointPropertiesSection.java

```diff
@@ -340,8 +305,11 @@ public IStatus validate(Object value) {
 				new UnsupportedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, p, page, getWidgetFactory()).create();
             // CLASS BASED PROPERTIES - REF OR CLASSNAMES AS STRINGS
             } else {
-				new ClassBasedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, prop, page, getWidgetFactory()).create();
+            	// for now we disable handling for this type - FUSETOOLS-2188
+//            	new ClassBasedParameterPropertyUICreatorForAdvanced(dbc, modelMap, eip, selectedEP, prop, page, getWidgetFactory()).create();
```

Comment:
we can remove it from this passage but I would keep the class itself as it will be of use once we improve the class type params handling.